### ISSUE TITLE
How does `editor_css` work with wysiwyg fields ?

### ISSUE THREAD

**Issue by jrfnl** [WebDevStudios/CMB2#679]
In the [field guide wiki page](https://github.com/WebDevStudios/CMB2/wiki/Field-Types#wysiwyg) it states that  `$field['options']['editor_css']` is `intended for extra styles for both visual and HTML editors buttons, needs to include the `<style>` tags, can use "scoped"`.

So I've tried passing it a style string, but it doesn't seem to have any effect.

```php
$my_cmb = new_cmb2_box( $this->get_box_definition() );
$my_cmb->add_field( array(
	'id'        => '_some_id_for_field',
	'name'  => __( 'Wysiwyg text field', 'my_plugin' ),
	'type'    => 'wysiwyg',
	'options'     => array(
		'media_buttons' => false, // show insert/upload button(s)
		'editor_css'    => '<style type="text/css" scoped="scoped">
	body#tinymce._some_id_for_field {
		font: 14px/1.5em Arial, Helvetica, sans-serif;
		color: #1b1c20;
	}
</style>',
	),
) );
```

I've tried to find some code examples, however, no luck so far. Also tried to search the CMB2 code base and can't find *any* reference whatsoever to `editor_css`, so I'm now actually wondering whether this has been implemented at all ?

**Comment by jtsternberg** [comment]
The wysiwyg field uses ['wp_editor'](https://codex.wordpress.org/Function_Reference/wp_editor) and passes the options array along to that function. 'editor_css' is an option for wp_editor.

**Comment by jrfnl** [comment]
Yeah, I looked into that, but still haven't been able to figure out why it's not working. If anyone has some working example code, it would be awesome if it could be added to the wiki.

**Comment by tw2113** [comment]
These two links describe the issue better than I ever could. The biggest issue is that the css gets rendered outside of the iframe, but needs to be inside. It is definitely an active one for CMB2 as well.

https://wordpress.stackexchange.com/questions/120831/how-to-add-custom-css-theme-option-to-tinymce/

https://wordpress.org/support/topic/customdynamic-css-in-tinymce

**Comment by jrfnl** [comment]
Thanks @tw2113 I'll have a read.
